conf/app.routes

```diff
@@ -37,3 +37,8 @@ GET         /check-company-registered-office-address      controllers.CheckCompa
 POST        /check-company-registered-office-address      controllers.CheckCompanyRegisteredOfficeAddressController.onSubmit(mode: Mode = NormalMode)
 GET         /edit/check-company-registered-office-address controllers.CheckCompanyRegisteredOfficeAddressController.onPageLoad(mode: Mode = CheckMode)
 POST        /edit/check-company-registered-office-address controllers.CheckCompanyRegisteredOfficeAddressController.onSubmit(mode: Mode = CheckMode)
+
+GET        /check-utr                        controllers.CheckUtrController.onPageLoad(mode: Mode = NormalMode)
```

Comment:
should we use url `/check-unique-taxpayer-reference` as [DDCYLS-3850](https://jira.tools.tax.service.gov.uk/browse/DDCYLS-3850) requested?